File: Linguaggi/Simulazioni/40/Python/Veicolo.py

class Veicolo:
    
    def __init__(self, codice, targa, modello, marca, costo):
        self.codice = codice
        self.targa = targa
        self.modello = modello
        self.marca = marca
        self.costo = costo
        
    def __str__(self):
        return str(self.targa) + "\t" +str(self.codice)+"\t"+self.modello+"\t"+self.marca+"\t"+str(self.costo)+"\t"
        
               

File: Linguaggi/Simulazioni/40/Java/Veicolo.java
public class Veicolo {
    protected int codice;
    protected String targa;
    protected String modello;
    protected String marca;
    protected double costo;

    public Veicolo(int codice, String targa, String modello, String marca, double costo) {
        this.codice = codice;
        this.targa = targa;
        this.modello = modello;
        this.marca = marca;
        this.costo = costo;
    }

    public String toString() {
        return this.targa+"\t"+
               this.codice+"\t"+
               this.modello+"\t"+
               this.marca+"\t"+
               this.costo;
    }
}
